EventPlanning.DAL/Address.cs

```diff
@@ -1,9 +1,47 @@
 ﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
 
 namespace EventPlanning.DAL
 {
     public class Address
     {
+        public string FullAddress()
```

Comment:
Does this output 
```
Street1
House
Number
Province
Country
```
? If yes it's not what would you expect. Most of the time (in my experience) there is no use case for this kind of string format.
Also, this can be easily unit tested.